### PR TITLE
[11.0][FIX] account_invoice_margin: Set purchase_price when user has not se invoice margin security group set

### LINKED ISSUE #52: Bug in account_invoice_margin
If you create an invoice without the **Show Invoice Margin** security group, the `purchase_price` field of `account.invoice.line` model isn't set.

**Steps to reproduce**:
1. With a user who doesn't have **Show Invoice Margin** permission
2. Create an invoice
3. Give permissions to the user to see the margins (Show Invoice Margin)
4. Press `ctrl+r` to reload
5. Check the invoice line. The margin percentage will be 100% and the cost will be 0.

imagen

Can it be related to the way the view is restricted to the group?

https://github.com/OCA/margin-analysis/blob/366c90b2f54901f4aff25098fb6a011f2adc4b2a/account_invoice_margin/views/account_invoice_margin_view.xml#L24-L25

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa
Fixes #52

### CODE REVIEW

**Comment by pedrobaeza** [comment]
As the field is not a many2one to other forbidden model, I think it's more convenient to have the field as invisible in main view, and have a new view only for the margin group that set the invisibility to 0, and thus you don't need any extra code.

**Comment by sergio-teruel** [comment]
Changes Done

File: account_invoice_margin/views/account_invoice_margin_view.xml [hunk 1]
```diff
@@ -2,48 +2,84 @@
 <!-- © 2017 Sergio Teruel <sergio.teruel@tecnativa.com>
      License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl-3). -->
 <odoo>
+    <!--
+    The xxx_to_store views only adds purchase price field in view to allow
```
Comment (pedrobaeza) [inline account_invoice_margin/views/account_invoice_margin_view.xml]:
```suggestion
    The xxx_to_save views only adds purchase price field in view to allow
```

**Review by pedrobaeza — APPROVED** [review]

**Comment by sergio-teruel** [comment]
Done!!

**Review by carlosdauden — APPROVED** [review]

**Pull request merged**